pages/api/agreements/index.js

```diff
@@ -0,0 +1,34 @@
+import { connectToDatabase } from '../../../util/mongodb';
+import AgreementsSchema from '../../../models/Agreements';
```

Comment:
NextJS nos permite usar rutas absolutas al baseUrl
```suggestion
import { connectToDatabase } from 'util/mongodb';
import AgreementsSchema from 'models/Agreements';
```